docs/README.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+
+Kernel Module Management also known as KMM is a Kubernetes operator which provides a simple way to manage, build, sign or deploy out-of-tree kernel modules, drivers or device plugins on selected nodes of an existing Kubernetes cluster.
+
+Once installed we will get a Deployment object consisting of a ReplicaSet with a pod acting as a controller manager in the new Namespace kmm-operator-system. 
+
+This controller provides the use of a new Kubernetes API object called Module which sets if the driver image already exists and has to be loaded, if it has to be in-cluster built (a ConfigMap should be provided to get this), if it will use a device plugin and also on which nodes would be deployed as a new DaemonSet will be created for each node’s kernel version matching the settings in a DriverContainer or device plugin within the Module definition.
+
+
+# Installation
+
+Installing the Kernel Module Management operator will create all necessary objects and permissions for the correct running of the operator:
+
+- A Namespace kmm-operator-system where manager will run.
+- Serviceaccount and RBAC permissions for KMM to operate.
+- The new CustomResourceDefinitions to add in the Kubernetes API.
+- The deployment of the manager itself.
+
+The following command will install the operator with all above necessary components:
+
+```bash
+kubectl apply -k https://github.com/kubernetes-sigs/kernel-module-management/config/default
+```
```

Comment:
I will suggest referring to https://github.com/kubernetes-sigs/kernel-module-management#getting-started instead of duplicating the command.

The main reason for not having duplication is that in the future we may update this command in the README.md and forget to update it here as well, hence, having an outdated doc.